app/scripts/modules/amazon/src/aws.settings.ts

```diff
@@ -1,13 +1,13 @@
 import { IProviderSettings, SETTINGS } from '@spinnaker/core';
 
-export interface IClassicLaunchWhitelist {
+export interface IClassicLaunchAllowlist {
   region: string;
   credentials: string;
 }
 
 export interface IAWSProviderSettings extends IProviderSettings {
   classicLaunchLockout?: number;
-  classicLaunchWhitelist?: IClassicLaunchWhitelist[];
+  classicLaunchAllowlist?: IClassicLaunchAllowlist[];
```

Comment:
Does anyone know who else uses this optional setting? I will tag them to inform them of the change.